grabbit/src/main/proto/node.proto

```diff
@@ -9,6 +9,7 @@ option java_outer_classname = "NodeProtos";
 message Node {
     required string name = 1;
     required Properties properties = 2;
+    repeated Node mandatoryChildNode = 3;
```

Comment:
also so make it multivalue the protobuffer generates method name and variable with `List` in it as `mandatoryChildNodeList` based on `repeated` flag.